pyshorteners/shorteners/bitly.py

```diff
@@ -63,16 +69,26 @@ def expand(self, url):
         Raises:
             ExpandingErrorException: If the API Returns an error as response
         """
-        expand_url = f'{self.api_url}v3/expand'
+        self.clean_url(url)
+        if url.startswith('http'):
+            url = url.replace('https://', '').replace('http://', '')
```

Comment:
currently I simply check if it starts with the protocol and remove it if needed
any suggestions for a better way to do it? I thought about regex or a dedicated function

Reply:
I might use urllib.parse (https://docs.python.org/3/library/urllib.parse.html).  Add `from urllib.parse import urlparse` to the top and `''.join(urlparse(url)[1:3])` here.

Reply:
sounds good
I'll replace update it ASAP